public/index.html

```diff
@@ -7,13 +7,13 @@
   <body>
     <div class='top-container'>
       <div id='singleItemPage'></div>
-      <div id='item-details'></div>
+      <!-- <div id='item-details'></div> -->
     </div>
-    <div id="sponsored-related-products"></div>
-    <div id="reviewsapp"></div>
-    <script  src='http://ec2-3-132-5-204.us-east-2.compute.amazonaws.com:3001/singleItemPage.js'></script>
-    <script  src='http://34.201.53.74:3002/dist/bundle.js'></script>
+    <!-- <div id="sponsored-related-products"></div>
+    <div id="reviewsapp"></div> -->
+    <script  src='http://localhost:3001/singleItemPage.js'></script>
+    <!-- <script  src='http://34.201.53.74:3002/dist/bundle.js'></script>
      <script  src='http://54.89.234.183:3003/bundle.js'></script>
-    <script  src='http://18.212.184.37:3004/bundle.js'></script>
+    <script  src='http://18.212.184.37:3004/bundle.js'></script> -->

```

Comment:
What is happening here? I thought you were meant to comment out all other services for now.

Reply:
I did